docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,304 @@
+    "from cleanlab.filter import find_label_issues\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import torchvision\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "\n",
+    "np.set_printoptions(suppress=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "144ad4c2-49bb-4147-a743-a83ed1656a11",
+   "metadata": {},
+   "source": [
+    "## 2. Get data, labels, and pred_probs\n",
+    "\n",
+    "In multilabel classification tasks, each token in the dataset is labeled with one or more of *K* possible classes.\n",
+    "To find label issues, cleanlab requires predicted class probabilities from a trained classifier. These `pred_probs` contain a length-*K* vector for **each** token in the dataset.  Here we use `pred_probs` which are out-of-sample predicted class probabilities for a subset of the [CelebA](https://www.kaggle.com/datasets/jessicali9530/celeba-dataset) dataset. Our example notebook [\"Training Multilabel classification for images\"](https://github.com/cleanlab/examples/blob/master/multilabel_classification/pytorch_network_training.ipynb) contains the code to produce such `pred_probs` and save them in a `.npz` file, which we simply load here. `data.csv` is a dataframe containing the labels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b5fa99a9-2583-4cd0-9d40-015f698cdb23",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred_probs = np.load(open('pred_probs.npy','rb'))\n",
+    "df = pd.read_csv(\"data.csv\").set_index(\"image_path\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f698518-5c1f-42e3-80b7-687ea286b233",
+   "metadata": {},
+   "source": [
+    "`pred_probs` is a list(of size K) of numpy arrays, representing the probability that a given value belongs to class K."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf83deac-f7e9-4a92-b498-a4e5fe67c6a3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class_names = list(df.columns)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0790e633-dd4b-4217-b32f-3e33ed56e739",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels=(df[class_names].to_numpy())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2227068f-4382-4e66-b2e4-4fc037223128",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def convert_multilabel_y_to_list(y):\n",
+    "    yl = list()\n",
+    "    for i in range(y.shape[0]):\n",
+    "        yl.append(np.where(y[i,:])[0].tolist())\n",
+    "    return yl    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "29ab0d31-5002-4935-8f96-3847edbc172c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multi_labels = convert_multilabel_y_to_list(labels)\n",
+    "\n",
+    "pred_labels = convert_multilabel_y_to_list(np.array((pred_probs>0.5),np.int32))"
+   ]
+  },
```

Comment:
show examples of ~3 well-labeled images from CelebA here (say first 3 in the dataset)

Reply:
Fixed: https://github.com/cleanlab/cleanlab/pull/517/commits/7aa53405e8abc5f3f0aa2f2a3a96f90f21516db1